### PR TITLE
QUAD 112 : Fixing bug on navbar currentpage and adding project to model

### PR DESCRIPTION
**Pull request by kavinya**
QUAD 112 : Fixing bug on navbar currentpage and adding project to model

### CODE REVIEW

File: Quadriga/.settings/org.eclipse.wst.common.component [hunk 1]
```diff
@@ -3,9 +3,7 @@
         <wb-resource deploy-path="/" source-path="/target/m2e-wtp/web-resources"/>
 		<wb-resource deploy-path="/" source-path="/src/main/webapp" tag="defaultRootSource"/>
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/java"/>
-        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/java"/>
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/resources"/>
-        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/resources"/>
 		<property name="java-output-path" value="target/classes"/>
         <property name="context-root" value="quadriga"/>
 	</wb-module>
```
Comment (jdamerow) [inline Quadriga/.settings/org.eclipse.wst.common.component:9]: don't commit this file.

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
I get the following error now:

```
javax.servlet.ServletException: Could not resolve view with name 'sites/projectblog' in servlet with name 'appServlet'
    org.springframework.web.servlet.DispatcherServlet.render(DispatcherServlet.java:1226)
    org.springframework.web.servlet.DispatcherServlet.processDispatchResult(DispatcherServlet.java:1027)
    org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:971)
    org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:893)
    org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:968)
    org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:859)
    javax.servlet.http.HttpServlet.service(HttpServlet.java:624)
    org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:844)
    javax.ser
```

Did you commit all changes? Seems like tiles-def entry is missing.

**Comment by kavinya** [comment]
@jdamerow  - Updated the contents of tiles-def and projectblog.jsp, which was missing in previous commit.

**Comment by jdamerow** [comment]
@kavinya , you are still committing a settings file.

**Comment by kavinya** [comment]
QUAD-112 : Closing this pull request as changes are included in QUAD-137.

**Pull request closed without merging**